File: libnd4j/include/dsp/runtime/bindings/java/examples/BasicUsage.java
import org.nd4j.dsp.runtime.SdxRuntime;

public class BasicUsage {
    public static void main(String[] args) {
        try (SdxRuntime runtime = SdxRuntime.create()) {
            System.out.println("ABI version: " + runtime.abiVersion());

            try (SdxRuntime.SdxModel model = runtime.loadModel("path/to/model.sdx", null)) {
                try (SdxRuntime.SdxContext ctx = model.createContext(null)) {
                    System.out.println("Runtime created successfully.");
                }
            }
        }
    }
}

File: libnd4j/include/dsp/runtime/bindings/csharp/examples/BasicUsage.cs
using System;
using Nd4j.Dsp.Runtime;

class BasicUsage
{
    static void Main(string[] args)
    {
        using var runtime = SdxRuntime.Create();
        Console.WriteLine($"ABI version: {runtime.AbiVersion()}");

        using var model = runtime.LoadModel("path/to/model.sdx");
        using var ctx = model.CreateContext();

        Console.WriteLine("Runtime created successfully.");
    }
}
